### ISSUE TITLE
[MU4 Issue] Cross staff problems after import mscz from 3.6.2. 

### ISSUE THREAD

**Issue by mart661331** [musescore/MuseScore#13644]
Cannot edit placement of cross staff beams, disappears.
Also some stems are not visible

ONLY after import

Windows 11, latest nightly

[test  cross staff edit problems after import from 3.6.2.mscz.zip](https://github.com/musescore/MuseScore/files/9726542/test.cross.staff.edit.problems.after.import.from.3.6.2.mscz.zip)

**Comment by oktophonie** [comment]
Please attach the original 3.6.2 file. The file above is a 4.0 file, in which I cannot reproduce the problem, and I cannot test how the migration process might affect the situation.

**Comment by mart661331** [comment]
[test import cross staf problem - kopie.zip](https://github.com/musescore/MuseScore/files/9732445/test.import.cross.staf.problem.-.kopie.zip)

the 3.6.2 mscz file
see for example measure 47

**Comment by oktophonie** [comment]
This works fine for me.
It imports like this:
![image](https://user-images.githubusercontent.com/20604032/194521096-0c66722e-bd43-4061-87e8-d2fed0a9195c.png)

And I can adjust however I like, and there are no problems with stems:
![image](https://user-images.githubusercontent.com/20604032/194521174-4b1d9111-cbab-498c-a8db-624253f86744.png)

This is confusing to me because the problem you are describing was fixed months ago, and I can't reproduce it now at all. Could you please confirm _exactly_ which build you are using? (Help > About, then hit the 'copy' button and paste your clipboard here. I get this, for example: "OS: Artix Linux, Arch.: x86_64, MuseScore version (64-bit): 4.0.0-3202248332, revision: github-musescore-musescore-56e2ae7")
![image](https://user-images.githubusercontent.com/20604032/194521462-72da8a4b-d8bd-442c-8bf1-766bef2607ab.png)

**Comment by mart661331** [comment]
It is strange. Now even the first staff of piano2, has the same
problem....measure 47.....

OS: Windows 10 Version 2009, Arch.: x86_64, MuseScore version (64-bit):
4.0.0-3194483041, revision: github-musescore-musescore-abc123456


Editie Windows 11 Home
Versie 22H2

Installatiedatum ‎06.‎okt.‎2022
Build van besturingssysteem 22621.608
Ervaring Windows Feature Experience Pack 1000.22634.1000.0

Op vr 7 okt. 2022 om 11:29 schreef Simon Smith ***@***.***>:

**Comment by cbjeukendrup** [comment]
(So that exact build can be downloaded from https://github.com/musescore/MuseScore/actions/runs/3194483041. For some reason, the commit information is not embedded properly in that build, so it says just `abc123456`...)

**Comment by mart661331** [comment]
What is the question?
Martin

Op vr 7 okt. 2022 om 12:09 schreef Casper Jeukendrup <
***@***.***>:

**Comment by cbjeukendrup** [comment]
Sorry, I had no question, I was just saying this for reference, since not everyone might know how to find the exact build if you know the version number but not the so-called "commit information".

**Comment by oktophonie** [comment]
I can't reproduce the problem with this exact build, on Windows, with the 3.6.2 score either. Maybe others will have more luck.

There are certainly some oddities with regard to incorrect beam placement in the score; I have a feeling https://github.com/musescore/MuseScore/pull/13423 may fix these, if it can ever be merged.

**Comment by mart661331** [comment]
I made a new score in 3.6.2 with cross staff.
imported in MU4. No problem. all works fine.

Then i loaded the imported score you have too, with the measure 47.

Then i copied measure 47 after the good working measure in the new imported
score.
Now the copied measure works fine, but the measure before, which worked
fine, has now a problem not being able to adjust placement... strange...

Op vr 7 okt. 2022 om 12:44 schreef Simon Smith ***@***.***>:

**Comment by mart661331** [comment]
Maybe it has something to do with import of xml file...
The scores with this problem I made from xml import.

**Comment by mart661331** [comment]
Hi,
I have tried a few things.

Problem starts always on the second page.

Even when I export from 3.6.2 to MIDI or MXL,  
Then import them in MU4, the same problems still exists.

**Comment by oktophonie** [comment]
Closing as duplicate of #13923
